onnxmltools/convert/xgboost/common.py

```diff
@@ -12,5 +12,9 @@ def get_xgb_params(xgb_node):
     else:
         # XGBoost < 0.7
         params = xgb_node.__dict__
-        
+
+    if ('n_estimators' not in params and
```

Comment:
no need to have ()

Reply:
I need to check because I'm not sure which versions of python support this. I use that because that's what's done in scikit-learn.

Reply:
It does not work without () for me.

Reply:
Is it due to parenthesis has to enclose the (not in)? not the full statement?

Reply:
The parenthesis are used to write the test over two lines, otherwise, I would have to use symbol \. Apparently, if ( … \n ...): seems to be the preferred one.